lib/vtls/openssl.c

```diff
@@ -2010,7 +2010,7 @@ static void ossl_session_free(void *sessionid, size_t idsize)
 {
   /* free the ID */
   (void)idsize;
-  SSL_SESSION_free(sessionid);
+  free(sessionid);
```

Comment:
Really? This looks like a rather drastic change that if necessary should rather be done in its own PR. Can you explain why/how you found/decided this?

Reply:
Eh sorry strike that. I see it now.